test6.py

```diff
@@ -0,0 +1,20 @@
+import torch
```

Comment:
Did you mean to add this file?

Reply:
No - sorry, fat fingered a commit I guess. My apologies.